crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -295,8 +294,6 @@ async fn sys_validation_workflow_inner(
         }
     }
 
-    // Allow unused mutable, because it's mutated when feature unstable-warrants is enabled.
-    #[allow(unused_mut)]
     let (mut summary, _invalid_ops, _forked_pairs) = workspace
```

Comment:
Now that it's always used
```suggestion
    let (mut summary, invalid_ops, _forked_pairs) = workspace
```